Directory.Build.props

```
@@ -1,6 +1,24 @@
 <Project>
+    <Sdk Name="ViHo.PackAsAnalyzer" Version="1.0.1" />
+
     <PropertyGroup>
         <Nullable>enable</Nullable>
+        <Version>1.11.0</Version>
```

```suggestion
        <Version>1.11.0-alpha1</Version>
```
Let's put this on a pre-release version, because the main branch build pipeline will auto-publish this once it merges.